### PR TITLE
Homework/4

### PR DESCRIPTION
**Pull request by Davydovychyurii**

### CODE REVIEW

File: HW4/src/getMin.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function getMin(a, ...b) {
```
Comment (AgentPrus) [inline HW4/src/getMin.js]: could you use instead 'a' and 'b' an 'arguments' object ?

File: HW4/src/isSmaller.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function isSmaller(a, b) {
+    return a < b ? true : false;
```
Comment (AgentPrus) [inline HW4/src/isSmaller.js]: you can remove trinariy expression and have the same result

File: HW4/src/isPrime.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+function isPrime(n) {
+    return Number.isInteger(n);
```
Comment (AgentPrus) [inline HW4/src/isPrime.js]: Try those numbers: 148, 150, 108; Output should be false.
You can see list of prime numbers here - https://ru.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%BF%D1%80%D0%BE%D1%81%D1%82%D1%8B%D1%85_%D1%87%D0%B8%D1%81%D0%B5%D0%BB

File: HW4/src/getMin.js (hunk 1 above)
Comment (Davydovychyurii) [reply HW4/src/getMin.js]: +

File: HW4/src/isSmaller.js (hunk 2 above)
Comment (Davydovychyurii) [reply HW4/src/isSmaller.js]: +

**Review by AgentPrus — APPROVED** [review]

File: HW4/src/isPrime.js (hunk 3 above)
Comment (Davydovychyurii) [reply HW4/src/isPrime.js]: +

File: HW4/src/getMin.js (hunk 1 above)
Comment (oksanadrul) [reply HW4/src/getMin.js]: @AgentPrus , perfect comment. I agree with you

**Pull request closed without merging**